src/Host.h

```diff
@@ -344,8 +348,11 @@ class Host : public QObject
     QString mCommandSeparator;
     bool mEnableGMCP;
     bool mEnableMSSP;
+    bool mEnableMSP;
     bool mEnableMSDP;
     bool mServerMXPenabled;
+    QString mMediaLocationGMCP;
+    QString mMediaLocationMSP;
```

Comment:
These variables hold the URLs for MSP and GMCP (Client.Media).  Because games could have legacy MSP and Client.Media running at the same time, it is two URLs, so there are no clashes between protocols.